lib/libfrr.h

```diff
@@ -35,6 +35,11 @@
 #define FRR_NO_CFG_PID_DRY		(1 << 3)
 #define FRR_NO_ZCLIENT		(1 << 4)
 
+enum frr_ret {
```

Comment:
Please remove this enum. "0 means OK" is standard procedure for any function returning `int`.